client/src/components/productOverview/StyleThumbnail.jsx

```diff
@@ -1,9 +1,28 @@
 import React from 'react'
+// import CheckCircle from '../reviews/CheckCircle.jsx';
+// use checkCircle later for selected style
+
+const StyleThumbnail = (props) => {
+
+  let thumbnail;
+  if (props.currentStyle.name === props.name) {
+    thumbnail =
+    <div className='style-thumbnail current-style-selection' onClick={props.updateStyle}>
+      <img src={props.thumbnail}></img>
+    </div>
+  } else {
+    thumbnail =
+    <div className='style-thumbnail non-current-style' onClick={props.updateStyle}>
+      <img src={props.thumbnail}></img>
+    </div>
+  }
+
+  return (
+    <>
```

Comment:
What does fragment do here?

Reply:
I think I was getting an error related to not wrapping the jsx return in a container. I didn't necessary want a div so I tried using a Fragment instead and it seemed to resolve the issue. Not sure if that is bad practice. Should I just make it a div?

Reply:
No. It's fine. I was just wondering why there would be an error. Because thumbnail is returning only one div. Will have to research about it. But this looks fine

Reply:
No. It's fine. I was just wondering why there would be an error. Because thumbnail is returning only one div. Will have to research about it. But this looks fine